core/server/api/authentication.js

```diff
@@ -8,6 +9,7 @@ var _                = require('lodash'),
     globalUtils      = require('../utils'),
     utils            = require('./utils'),
     errors           = require('../errors'),
+    models           = require('../models'),
```

Comment:
This is already included above with the name `dataProvider`, which is something very old. Would be good to fix this.